lib/modules/manager/sbt/index.ts

```diff
@@ -13,6 +16,6 @@ export const supportedDatasources = [
 ];
 
 export const defaultConfig = {
-  fileMatch: ['\\.sbt$', 'project/[^/]*.scala$'],
+  fileMatch: ['project/.*\\.scala$', '\\.sbt$', 'project/[^/]*.scala$'],
```

Comment:
Can this be combined with the third one? Is directory nesting a common practice in `project/` directory?